### PR TITLE
Static routes for networks

### PR DESCRIPTION
**Pull request by inercia**
Static routes for networks. Users could now specify static routes in a network like this:

```hcl
    ...
    # (Optional) one or more static routes.
    # "cidr" and "gateway" must be specified. The format is:
    routes = [
    {
        cidr = "10.17.0.0/16"
        gateway = "10.18.0.2"
    },
    ...
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,12 +11,29 @@ build: gofmtcheck golint vet
...
+# unit tests
+# usage:
+# - run all the unit tests: make test
+# - run some particular test: make test TEST_ARGS="-run TestAccLibvirtDomain_Cpu"
 test:
-	go test -v -covermode=count -coverprofile=profile.cov ./libvirt
+	go test -v $(TEST_ARGS_DEF) $(TEST_ARGS) ./libvirt
 	go test -v .
+
+# acceptance tests
```
Comment (MalloZup) [inline Makefile]: nice one here. can you update also this doc: https://github.com/dmacvicar/terraform-provider-libvirt/blob/master/CONTRIBUTING.md#running-the-tests-testacc

Comment (inercia) [reply Makefile]: Done.

**Comment by inercia** [comment]
@MalloZup I have updated the PR, implementing the `Read` part so Terraform can know about the current static routes. I don't really know much about how Terraform deals with `Update`, but I guess Terraform will know if things need to be updated when it can `Read` stuff, right?

**Comment by MalloZup** [comment]
@inercia thx i will have look later on this.and let u know :dog: :national_park:

**Comment by inercia** [comment]
It seems routes cannot be `Update`d, so we can skip our concerns about not implementing that part...

**Comment by dmacvicar** [comment]
So, given that we use data sources for other network configurations, was that evaluated for this feature?

**Comment by MalloZup** [comment]
Thank you for remark, 🦄 I opened an issue regarding this https://github.com/dmacvicar/terraform-provider-libvirt/issues/539

**Pull request merged**